src/components/ConnectionError/ConnectionError.test.tsx

```diff
@@ -38,4 +40,20 @@ describe("ConnectionError", () => {
     await userEvent.click(screen.getByRole("button", { name: "refreshing" }));
     expect(window.location.reload).toHaveBeenCalled();
   });
+
+  it("displays Audit Logs errors", async () => {
+    const state = rootStateFactory.withGeneralConfig().build({
+      juju: { auditEvents: { errors: "Oops!" } },
+    });
+    const {
+      result: { rerender },
+    } = renderComponent(<ConnectionError />, { state });
```

Comment:
You can avoid the rerender by rendering the Toaster initially. This is how we handle this case in other tests.

```suggestion
    renderComponent(<>
        <Toaster />
        <ConnectionError />
      </>, { state });
```

Reply:
Done!